releasenotes/notes/sabre-sort-rng-056f26f205e38bab.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Internal tweaks to the routing algorithm in :class:`.SabreSwap`, used in
+    transpilation of non-dynamic circuits at all non-zero optimization levels,
+    have sped up routing for very large circuits.  As an example, the time to
+    route a depth-5 :class:`.QuantumVolume` circuit for a 1081-qubit heavy-hex
+    coupling map is approximately halved.
```

Comment:
I'd probably add an upgrade note here about results changing for fixed seed. While we don't explicitly guarantee seed for see compatibility between releases. Documenting this as a potential upgrade note seems like the right thing to do (I also think we've done this in previous releases when changing RNG behavior).

Reply:
Huh, I actually thought I'd written that, but I guess not!  Done in 60ddde195.